typegraph/typegraph/providers/prisma/type_generator.py

```diff
@@ -124,6 +122,42 @@ def get_where_type(
         else:
             return t.struct(fields).named(name)
 
+    def deep_map(self, tpe: t.typedef, fn: callable, seen=set()) -> t.struct:
```

Comment:
What about `t.boolean`?

Reply:
I have already removed the `if` condition, it was merely a test to see if it worked. Terminal nodes should be automatically handled in the last return statement (`t.boolean`, `t.string`, ..etc).